shared-libs/transitions/src/transitions/user_replace.js

```diff
@@ -162,16 +90,19 @@ module.exports = {
       throw new Error(`Configuration error. Token login must be enabled to use the user_replace transition.`);
     }
   },
-  filter: (doc, info = {}) => {
-    return doc.form === 'replace_user'
-      && !transitionUtils.hasRun(info, NAME);
+  filter: (doc) => {
+    const contactType = contactTypeUtils.getContactType(config.getAll(), doc);
+    if (!contactType || !contactTypeUtils.isPersonType(contactType)) {
+      return false;
+    }
+
+    return doc.replaced && doc.replaced.status === 'READY';
   },
   onMatch: change => {
     return replaceUser(change.doc)
-      .then(() => {
-        return true;
-      })
+      .then(() => true)
       .catch(err => {
+        change.doc.replaced.status = 'ERROR';
```

Comment:
What do you think of using an enum-like object for these status constants? Just like in `user-replace.service.ts`

```js
const ReplaceStatus = {
  PENDING: 'PENDING', // Waiting on sync to complete
  READY: 'READY' // Ready to be replaced
  COMPLETE: 'COMPLETE', // The new user has been created
  ERROR: 'ERROR', // The new user could not be created
}
```